package.json

```diff
@@ -5,28 +5,32 @@
   "main": "index.js",
   "scripts": {
     "eslint": "eslint .",
-    "test": "npm run eslint"
+    "test": "npm run eslint",
+    "prettier": "prettier --config .prettierrc '**/*.{js,json}'"
   },
   "repository": "blogfoster/eslint-config-blogfoster",
-  "keywords": [
-    "eslint"
-  ],
+  "keywords": ["eslint"],
   "author": "blogfoster GmbH",
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/blogfoster/eslint-config-blogfoster/issues"
   },
   "homepage": "https://github.com/blogfoster/eslint-config-blogfoster#readme",
   "dependencies": {
-    "babel-eslint": "^7.2.3",
-    "eslint-config-airbnb": "^15.0.1",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-mocha": "^4.9.0",
-    "eslint-plugin-react": "^7.1.0",
-    "estraverse-fb": "^1.3.1"
+    "babel-eslint": "^8.0.0",
+    "eslint": "^4.7.2",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-config-prettier": "^2.6.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-mocha": "^4.11.0",
+    "eslint-plugin-react": "^7.4.0"
   },
   "devDependencies": {
-    "eslint": "^3.19.0"
+    "eslint": "^4.7.2",
+    "prettier": "^1.7.0"
+  },
+  "peerDependencies": {
+    "eslint": ">= 4"
```

Comment:
did you check if eslint for and the current rules work with our projects? last time i tried it didn't

Reply:
Er wait, why is eslint a peer dep now actually?

Reply:
@sfrdmn i think, so that you get a npm error while installing just this package from out-side?

Reply:
Ah right, makes sense. I forget we were in the config repo for a second :P

Reply:
@5c077yP I didn't yet test this but it's on my Todos's 😉